sanic/request.py

```diff
@@ -135,36 +110,43 @@ def __repr__(self):
         class_name = self.__class__.__name__
         return f"<{class_name}: {self.method} {self.path}>"
 
-    def body_init(self):
-        """.. deprecated:: 20.3
-        To be removed in 21.3"""
-        self.body = []
-
-    def body_push(self, data):
-        """.. deprecated:: 20.3
-        To be removed in 21.3"""
-        self.body.append(data)
-
-    def body_finish(self):
-        """.. deprecated:: 20.3
-        To be removed in 21.3"""
-        self.body = b"".join(self.body)
+    async def respond(
+        self, response=None, *, status=200, headers=None, content_type=None
```

Comment:
Need to take a closer look to confirm that this is actually the correct place for this.